### PR TITLE
add details on default value for load balancing algorithm

### PR DESCRIPTION
**Pull request by yomovh**
please duplicate to fr / en

### CODE REVIEW

File: pages/public_cloud/public_cloud_network_services/getting-started-01-create-lb-service/guide.fr-fr.md [hunk 1]
```diff
@@ -64,7 +64,12 @@ L'interface vous informera si le réseau privé ou le sous-réseau ne sont pas c
...
 - Choisissez ensuite le type de moniteur d'intégrité (Health Monitor). Notez que certains types de health monitors n'étant pas compatibles avec certains protocoles, l'interface utilisateur filtre ces types afin que vous puissiez uniquement choisir des éléments compatibles. Pour plus d'informations sur la compatibilité du health monitor, consultez [cette page](/pages/public_cloud/public_cloud_network_services/concepts-01-public-cloud-networking-concepts).
-- Enfin, parmi les instances de votre région, choisissez l'IP et le port membre qui feront partie du pool. Notez que pour simplifier le processus de configuration, votre pool doit avoir le même protocole que le listener, et que le membre ne peut être choisi qu'à partir de l'instance. Ces limitations peuvent être contournées en ignorant cette partie de la configuration et en utilisant la configuration du pool/membre une fois le Load Balancer créé. 
+- Enfin, parmi les instances de votre région, choisissez l'IP et le port membre qui feront partie du pool.
```
Comment (Kazarus4) [inline pages/public_cloud/public_cloud_network_services/getting-started-01-create-lb-service/guide.fr-fr.md]:
```suggestion
- Enfin, parmi les instances de votre région, choisissez l'adresse IP et le port membre qui feront partie du pool.
```

File: pages/public_cloud/public_cloud_network_services/getting-started-01-create-lb-service/guide.fr-fr.md [hunk 2]
```diff
@@ -64,7 +64,12 @@ L'interface vous informera si le réseau privé ou le sous-réseau ne sont pas c
...
+
+> [!primary]
```
Comment (Kazarus4) [inline pages/public_cloud/public_cloud_network_services/getting-started-01-create-lb-service/guide.fr-fr.md]:
```suggestion
> [!primary]
>
```

File: pages/public_cloud/public_cloud_network_services/getting-started-01-create-lb-service/guide.fr-fr.md [hunk 3]
```diff
@@ -64,7 +64,12 @@ L'interface vous informera si le réseau privé ou le sous-réseau ne sont pas c
...
-- Enfin, parmi les instances de votre région, choisissez l'IP et le port membre qui feront partie du pool. Notez que pour simplifier le processus de configuration, votre pool doit avoir le même protocole que le listener, et que le membre ne peut être choisi qu'à partir de l'instance. Ces limitations peuvent être contournées en ignorant cette partie de la configuration et en utilisant la configuration du pool/membre une fois le Load Balancer créé. 
+- Enfin, parmi les instances de votre région, choisissez l'IP et le port membre qui feront partie du pool.
+
+> [!primary] 
+> Pour simplifier le processus de configuration, votre pool doit avoir le même protocole que le listener et le membre ne peut être choisi qu'à partir d'une instance. De plus, l'algorithme de load balancing est par défaut `ROUND_ROBIN`.
```
Comment (Kazarus4) [inline pages/public_cloud/public_cloud_network_services/getting-started-01-create-lb-service/guide.fr-fr.md]:
```suggestion
> Pour simplifier le processus de configuration, votre pool doit avoir le même protocole que le listener et le membre ne peut être choisi qu'à partir d'une instance. De plus, l'algorithme de load balancing par défaut est : `ROUND_ROBIN`.
```

**Review by Kazarus4 — APPROVED** [review]

**Review by bs-ovh — APPROVED** [review]

**Comment by tcpdumpfbacke** [comment]
Thank you @yomovh !

**Pull request merged**